frontend/packages/akr/src/components/clerkTranslator/authorisation/AuthorisationFields.tsx

```diff
@@ -30,7 +30,7 @@ import { AuthorisationUtils } from 'utils/authorisation';
 interface AuthorisationFieldsProps {
   authorisation: Authorisation;
   setAuthorisation: (
-    a: ((prevState: Authorisation) => Authorisation) | Authorisation
+    a: ((prevState: Authorisation) => Authorisation) | Authorisation,
```

Comment:
Näitä loppupilkkuja on tässä PR:ssä lisätty aika runsaasti. Onko nää kenties seurausta jonkun linttaus-depsun päivityksestä tms? Tuottaa aika paljon ylimääräistä hälyä ja voi ehkä joku merkityksellinen muutos jäädä huomaamatta sen seurauksena.

Reply:
Joo, lintterin päivityksestä. Ehkä defaultit muuttunu lintterissä päivityksen yhteydessä. Katson jos saisin jollain configilla tämän säädettyä samalla tavalla kuin aiemmin.

Reply:
Jees. Yksi vaihtoehtohan voisi olla jättää toi lintterin päivitys pois tästä PR:stä ja päivittää se erillisessä PR:ssä tyylimuutoksineen.

Reply:
Tai no, ehkä voi jäädä myös tähän nykyiseen kuosiin. Taisin saada selailtua kaikki filut läpi.

Reply:
Äh. Kokeilinpa testata näitä muutoksia lokaalisti ja näköjään ainakin mulla prettier valittaa noista ylimääräisistä pilkuista. Ajoin `yarn install` ja sen jälkeen `yarn yki:start:dev-server`. Jäiköhän tästä joku askel nyt tekemättä?

Reply:
Onko 20.x versio nodesta? Ja ainakin mulla tuntui välillä vaativan, että poistaa node_modules tuolta frontend alta.

Reply:
Joo, toi node_modules poistaminen taisi taas kerran olla avain onneen.